src/Components/Nav/Nav.js

```diff
@@ -1,2 +1,128 @@
-import React from "react";
-import ReactDOM from "react-dom";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { AiOutlineBell } from "react-icons/ai";
+import styled from "styled-components";
+
+let controlActivation = [
+  { id: 1, path: "/Today", content: "투데이" },
+  { id: 2, path: "/Search", content: "검색" },
+  { id: 3, path: "/Feeds", content: "피드" },
+  { id: 4, path: "/MyBooks", content: "내서재" },
+  { id: 5, path: "/", content: "관리" },
+];
+
+function Nav() {
+  let [menu, setMenu] = useState(1);
+  let [logout, setlogout] = useState(false);
+
+  setlogout = () => {
+    localStorage.removeItem("Authorization");
+  };
+
+  return (
+    <NavBar>
+      <CommonContainer>
+        <MainMenu>
+          <Link to="/">
+            <img src="/images/gnb_logo.png" alt="윌리의 서재 로고" />
+          </Link>
+          <ul className="clearfix">
+            {controlActivation.map((tag, idx) => {
+              return (
+                <MenuList
+                  key={idx}
+                  menu={menu}
+                  active={tag.id === menu}
+                  onClick={() => {
+                    setMenu(tag.id);
+                  }}
+                >
+                  {tag.content}
+                </MenuList>
+              );
+            })}
+          </ul>
+        </MainMenu>
+        <UserAction>
+          <Action alarm="alarm">
+            <AiOutlineBell />
+          </Action>
+          <Action>
+            <BlackBtn
+              onClick={() => {
+                setlogout(true);
+              }}
+            >
+              로그아웃
+            </BlackBtn>
+          </Action>
+        </UserAction>
+      </CommonContainer>
+    </NavBar>
+  );
+}
+
+const BlackBtn = styled.button`
+  display: inline-block;
+  padding: 7px 17px;
+  background: #333;
+  border-radius: 4px;
+  font-size: 13px;
+  color: #fff;
+`;
+
+const NavBar = styled.nav`
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  background: #fff;
+  border-bottom: 1px solid #eee;
+`;
+
+const CommonContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1280px;
+  height: 64px;
+  margin: 0 auto;
+`;
+
+const MainMenu = styled.div`
+  display: flex;
+  justify-content: start;
+  align-items: center;
+
+  a {
+    display: inline-block;
+    width: 63px;
+    height: auto;
+    margin: 4px 25px 0 0;
+
+    img {
+      width: 100%;
+    }
+  }
+`;
+
+const MenuList = styled.li`
+  float: left;
+  padding: 12px 20px;
+  font-size: 20px;
+  cursor: pointer;
+  font-weight: ${(props) => props.active && "600"};
```

Comment:
```suggestion
  font-weight: ${({ active }) => active && "600"};
```
함수 파라미터 자리에서 직접 구조분해할당할 수 있습니다

Reply:
✔ 수정했습니다!